compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -117,7 +117,7 @@ macro_rules! maybe_recover_from_interpolated_ty_qpath {
     };
 }
 
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
Is this needed for something? Otherwise, can you remove it?

Reply:
Ah, I was using this for debugging, so thought this might be useful. Anyway, I'll remove it. Thanks!